### PR TITLE
Fixes lack of docker environment.

### PR DESCRIPTION
**Pull request by stefanoborini**
Fixes lack of docker environment due to change in HOME dir
and associated incorrect configuration/spawning of the singleuserapp.

### CODE REVIEW

**Comment by stefanoborini** [comment]
We don't test on mac, so that code cannot be hit.

File: remoteappmanager/spawner.py [hunk 1]
```diff
@@ -185,3 +187,13 @@ def _clean_up_workspace_dir(self):
                                self._virtual_workspace, str(exception))
             else:
                 self.log.info('Removed %s', self._virtual_workspace)
+
+
+def _docker_envvars():
+    """Returns a dictionary containing the docker environment variables, if
+    present. If not present, returns an empty dictionary"""
+    env = {}
+    for envvar in ["DOCKER_HOST", "DOCKER_CERT_PATH", "DOCKER_MACHINE_NAME"]:
```
Comment (kitchoi) [inline remoteappmanager/spawner.py]: `DOCKER_TLS_VERIFY` is a relevant docker environment variables too.

Comment (kitchoi) [inline remoteappmanager/spawner.py]: Minor comment: dict comprehension?

**Comment by kitchoi** [comment]
LGTM, please merge when green.

**Pull request merged**